qa/L0_infer/test.sh

```diff
@@ -63,6 +63,9 @@ if [ "$TEST_VALGRIND" -eq 1 ]; then
     LEAKCHECK_ARGS_BASE="--leak-check=full --show-leak-kinds=definite --max-threads=3000 --num-callers=20"
     SERVER_TIMEOUT=4000
     rm -f $LEAKCHECK_LOG_BASE*
+    # Remove onnx and python backends for now as the server hangs up when 
```

Comment:
Did you create follow-up tickets to investigate this? If yes, then can you add the DLIS-<IDs> here?

Reply:
Added ticket number.